packages/commonwealth/client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -1416,7 +1421,46 @@ const ViewProposalPage: m.Component<
                   vnode.state.polls?.map((poll) => [poll.id, poll])
                 ).values(),
               ].map((poll) => {
-                return m(ProposalPollCard, { poll, thread: proposal });
+                return m('.poll-card-wrapper', {}, [
+                  m(PollCard, {
+                    pollType: PollType.Offchain,
+                    multiSelect: false,
+                    pollEnded:
+                      poll.endsAt && poll.endsAt?.isBefore(moment().utc()),
+                    hasVoted:
+                      app.user.activeAccount &&
```

Comment:
this is all fine right now, but we should begin thinking about a polls controller that stores all this data and then the poll cards can just take in an id and populate based on modeled data. This is for a separate PR in the future, but wanted to plant the seed 🌱

Reply:
🚀